common/desktop-exports

```diff
@@ -144,3 +144,9 @@ if [ ! -f $DCONF_DEST_USER_DIR/user ]; then
     ln -s /home/$USER/.config/dconf/user $DCONF_DEST_USER_DIR
   fi
 fi
+
+# Testability support
```

Comment:
Thanks for the review. I could make it conditional but the testability/ directory will always be available, it will have the libraries or not depending on whether you connected to the interface provided by autopilot-qt or not. 
Autopilot is a Qt and Gtk solution, that's why I include this here, however, I can move this to the qt and gtk launchers